src/Microsoft.Diagnostics.Monitoring.EventPipe/DiagnosticsEventPipeProcessor.cs

```diff
@@ -13,38 +13,42 @@
 using Graphs;
 using Microsoft.Diagnostics.NETCore.Client;
 using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Extensions;
 using Microsoft.Diagnostics.Tracing.Parsers.Clr;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.Diagnostics.Monitoring
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
 {
-    public enum PipeMode
-    {
-        Logs = 1,
-        Metrics,
-        GCDump,
-        ProcessInfo
-    }
-
-    public class DiagnosticsEventPipeProcessor : IAsyncDisposable
+    internal partial class DiagnosticsEventPipeProcessor : IAsyncDisposable
```

Comment:
I can certainly make that change, now that DiagnosticsEventPipeProcessor is just an implementation detail. This change does add all the pipelines but they all delegate to DiagnosticsEventPipeProcessor. This would be a separate PR.